packages/js/src/decorator/gutenberg.js

```diff
@@ -350,10 +366,15 @@ function createAnnotations( html, richTextIdentifier, attribute, block, marks )
 	const text = record.text;
 
 	return flatMap( marks, ( ( mark ) => {
-		const annotations = calculateAnnotationsForTextFormat(
-			text,
-			mark
-		);
+		let annotations;
+		if ( marks[ 0 ].hasBlockPosition && marks[ 0 ].hasBlockPosition() ) {
```

Comment:
Is there a specific reason why the check uses `marks[ 0 ].hasBlockPosition` instead of using `mark.hasBlockPosition` ? Because at this point we're already looping over `marks` right? 🤔

Reply:
Oh, that's a little embarrassing then 🙈 . I will change it and see what happens.

Reply:
@hdvos was this resolved, shall it stay as it is now? :)

Reply:
This is resolved. The new check can be seen in [`getAnnotationsHelpers.js` ](https://github.com/Yoast/wordpress-seo/blob/73a533b0024fcb0a0b0a7141e8a7ea31dfdcb33d/packages/js/src/decorator/helpers/getAnnotationsHelpers.js#L30)